impeller/renderer/backend/metal/surface_mtl.mm

```diff
@@ -188,6 +188,15 @@
       ContextMTL::Cast(context.get())->CreateMTLCommandBuffer();
   [command_buffer commit];
   [command_buffer waitUntilScheduled];
+#else
```

Comment:
Can we just move this into its own method to dry this up? We can move the comments and such to that method too.

```
if (ShouldWaitUntilScheduled()) {
  [command_buffer commit];
  [command_buffer waitUntilSh...];
}
```

Reply:
Done